azure-devops/azext_devops/dev/repos/_help.py

```diff
@@ -48,3 +42,55 @@ def load_repos_help():
     short-summary: Manage Git repositories import
     long-summary:
     """
+
+    helps['repos policy'] = """
+    type: group
+    short-summary: Manage branch policy.
+    long-summary:
+    """
+
+    create_update_helptext = """
+        - name: {0} an approver count policy
+          text: |
+            az repos policy {1} --branch "refs/heads/master" -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type ApproverCountPolicy --allowDownvotes False --creatorVoteCounts False --minimumApproverCount 2 --resetOnSourcePush True --isBlocking --isEnabled
+
+        - name: {0} a build policy
+          text: |
+            az repos policy {1} --branch "refs/heads/master" -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type BuildPolicy --buildDefinitionId 72 --displayName Build_Required --manualQueueOnly False --queueOnSourceUpdateOnly True --validDuration 12 --isBlocking --isEnabled
+
+        - name: {0} a file size policy
+          text: |
+            az repos policy {1} -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type FileSizePolicy --maximumGitBlobSizeInBytes 5012 --useUncompressedSize True --isBlocking --isEnabled
+
+        - name: {0} a merge strategy policy
+          text: |
+            az repos policy {1} --branch "refs/heads/master" -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type MergeStrategyPolicy --useSquashMerge True --isBlocking --isEnabled
+
+        - name: {0} a comment requirement policy
+          text: |
+            az repos policy {1} --branch "refs/heads/master" -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type CommentRequirementsPolicy --isBlocking --isEnabled
+
+        - name: {0} a work item linking policy
+          text: |
+            az repos policy create --branch "refs/heads/master" -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type WorkItemLinkingPolicy --isBlocking --isEnabled
+
+        - name: {0} a required reviewer policy
+          text: |
+            az repos policy create --branch "refs/heads/master" -r ac6b3157-6af1-4afa-b9d5-80d9ed3afd72 --policy-type RequiredReviewersPolicy --message this_is_policy_message --requiredReviewerIds gsaral@microsoft.com;atbagga@microsoft.com --isBlocking --isEnabled
+    """
+
+    helps['repos policy create'] = """
+    type: command
+    short-summary: Create a policy.
+    long-summary: see https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies-overview?view=azure-devops for more details
```

Comment:
Caps S. 

The link to docs was remove earlier from one such help text from what I remember. 
I would say it should be there but just mentioning since it was removed last time.

Reply:
yup remember that, and then when we got feedback from a user we said "makes sense".
I can totally tell no one will be able to decode the response json without a good documentation and copy pasting whole API doc here is useless.
https://teams.microsoft.com/l/message/19:3dc2223b84c041848abd68b914b323b3@thread.skype/1548664020935?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=f0fb1404-b891-4ef9-becc-eb58cba47928&parentMessageId=1548368816780&teamName=Azure%20DevOps%20CLI%20Feedback&channelName=General&createdTime=1548664020935